### PR TITLE
merge PR 477 from 1.0

### PR DESCRIPTION
**Pull request by dkent600**
Merges https://github.com/daostack/arc.js/pull/477 from 1.0

### CODE REVIEW

File: src/plugins/fundingRequest/plugin.ts [hunk 1]
```diff
@@ -138,7 +138,7 @@ export class FundingRequest
       method: 'propose',
       args: [
         options.beneficiary,
-        options.amount.toNumber(),
+        options.amount.toString(),
```
Comment (orenyodfat) [inline src/plugins/fundingRequest/plugin.ts:141]: this plugin is not supported in alchemy yet. so why update that ?
@jellegerbrandy is that needed for Common ?

File: src/plugins/genericPlugin/plugin.ts [hunk 2]
```diff
@@ -43,7 +44,7 @@ export class GenericPlugin extends ProposalPlugin<
   IGenericPluginState,
   IGenericPluginProposalState,
   IProposalCreateOptionsGS
-> {
+  > {
```
Comment (orenyodfat) [inline src/plugins/genericPlugin/plugin.ts:47]: why additiional space is here ?

Comment (dkent600) [reply src/plugins/genericPlugin/plugin.ts:47]: This was generated by my editor. I'm not sure what rules it is following, but the change makes sense by aligning the bracket with the lines above.  Looks clean and harmless enough to me.

File: src/plugins/fundingRequest/plugin.ts (hunk 1 above)
Comment (dkent600) [reply src/plugins/fundingRequest/plugin.ts:141]: the change is needed in any case.  without it we would have a bug by casting a BN to a number, assuming the BN can easily overflow a `number`.

Comment (jellegerbrandy) [reply src/plugins/fundingRequest/plugin.ts:141]: you are asking me if the funding request plugin is needed for common?

yes, it is.

Comment (jellegerbrandy) [reply src/plugins/fundingRequest/plugin.ts:141]: Yeah, this is good 👍

**Pull request merged**